explanation.md

```diff
@@ -1,74 +1,52 @@
-[Version française](./explanation_fr.md)
+[Lire en version française](./explanation_fr.md)
 
 # Methodology
 
-Graphgenerator can recreate the network of interactions of Twitter accounts on a given hashtag or word (or group of words) 
-given. 
+Graphgenerator can recreate the network of interactions of Twitter accounts on a given hashtag or word (or group of words) given. 
 
 ## Collecting tweets
 
-The data is collected by `Graphgenerator` using the scraping tool `snscrape`. This tool collects the data
-that is displayed when you search using the Twitter search bar. It only allows to retrieve 
-the retweets of the last seven days (beyond that, only tweets and quotes can be collected). The 
-collection is limited to the last seven days.
+The data is collected by Graphgenerator using the scraping tool `snscrape`. This tool collects the data that is displayed when you search using the Twitter search bar. It only allows to retrieve the retweets of the last seven days (beyond that, only tweets and quotes can be collected). The collection is limited to the last seven days.
```

Comment:
Remove last sentence as it is redundant, and use bold

The data is collected by Graphgenerator using the scraping tool `snscrape`. This tool collects the data that is displayed when you search using the Twitter search bar. It only allows to retrieve the retweets of the **last seven days** (beyond that, only tweets and quotes can be collected).